publisher/product/datadictionary/build.sh

```diff
@@ -53,18 +53,21 @@ function publishProductDataDictionaryContent() {
   local maturityLevel="$(yq '.ontologies.maturity_level_definition.[] | "--maturity-level " + .iri + "=" + .label' < \
 	"${source_family_root}/etc/onto-viewer-web-app/config/ontology_config.yaml" 2>/dev/null | xargs echo -n)"
 
+  local maturityLevelProperty="$(yq '.ontologies | "--maturity-level-property " + .maturity_level_property' < \
```

Comment:
This should have `--maturity_level_property` instead of `--maturity-level-property` - see:
 'https://github.com/edmcouncil/onto-viewer/blob/d24cd2d8e3d2f7d211ec15651c8cf6939c96cef8/onto-viewer-config-loader/src/main/resources/default_ontology_config.yaml#L12'.